x/oracle/keeper/upgrade.go

```diff
@@ -58,3 +58,74 @@ func (k Keeper) ApplyUpgrade(ctx sdk.Context, info *types.OracleUpgradeInfo) err
 	ctx.Logger().Info("Oracle upgrade was successful.", fmt.Sprintf("uniqueID: %s, height: %v", info.UniqueId, info.Height))
 	return nil
 }
+
+func (k Keeper) UpgradeOracle(ctx sdk.Context, msg *types.MsgUpgradeOracle) error {
```

Comment:
Why don't we do duplication check for upgrade as done in #582?

Reply:
Can you tell me what kind of duplication check it is? Thanks

Reply:
I meant that let's check if the oracle has already requested upgrade.

The exact link for example is:
https://github.com/medibloc/panacea-core/blob/1c69a6de0ebc97888b33d92718875dc523c84cac/x/oracle/keeper/oracle.go#L20

Maybe,
```go
existing, err := k.GetOracleUpgrade(ctx, msg.GetUniqueId(), msg.GetOracleAddress())
// check already requested upgrade
if existing {
    return fmt.Errorf("already requested upgrade")
}
```

Reply:
Okay, thanks. I fixed it.